GitUI/Translation/French.xlf

```diff
@@ -9202,7 +9202,7 @@ when all items are shown:</source>
       <trans-unit id="lblMenuEntries.Text">
         <source>Select items to be shown in the cascaded context menu.
 (Unchecked items will be shown top level for direct access.)</source>
-        <target>Sélectionneé les items à afficher dans
+        <target>Sélectionnée les items à afficher dans
```

Comment:
It should either be `Sélectionner les ...` or `Sélectionnez les ....` (depending on the style used in the rest of the translation).

Also, `items` is an english loan word and should also be translated. It should be either `entrées` (entries) ou `options` as in `Sélectionnez les entrées à afficher dans le menu contextuel en cascade` or `Sélectionnez les options à afficher dans le menu contextuel en cascade`. 

Wikipedia for [Menu contextuel](https://fr.wikipedia.org/wiki/Menu_contextuel) uses the term `option`. I guess we would need the opinion of one more french speaking person to decide :)